extensions/wikia/WikiFeatures/templates/WikiFeaturesSpecial_index.php

```diff
@@ -1,36 +1,36 @@
 <section class="WikiFeatures" id="WikiFeatures">
 	<h2 class="heading-features">
-		<?= wfMsg('wikifeatures-heading') ?>
+		<?= wfMessage( 'wikifeatures-heading' )->escaped(); ?>
 	</h2>
 	<p class="creative">
-		<?= wfMsg('wikifeatures-creative') ?>
+		<?= wfMessage( 'wikifeatures-creative' )->escaped(); ?>
 	</p>
-	
+
 	<ul class="features">
-		<? foreach ($features as $feature) { ?>
-			<?= F::app()->getView( 'WikiFeaturesSpecial', 'feature', array('feature' => $feature, 'editable' => $editable ) ) ?>
-		<? } ?>
+		<? foreach ( $features as $feature ): ?>
+			<?= $app->renderPartial( 'WikiFeaturesSpecial', 'feature', [ 'feature' => $feature, 'editable' => $editable ] ); ?>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "editable" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "WikiFeaturesSpecial" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "feature" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)